docs/create-service-principal.md

```diff
@@ -9,6 +9,8 @@ This topic shows you how to permit a service principal (such as an automated pro
 Install and configure Azure CLI following the documentation [**HERE**](http://azure.microsoft.com/en-us/documentation/articles/xplat-cli/).
 
 **NOTE:**
+
+* If you already have npm / Node.js installed, run `npm install azure-cli -g`
```

Comment:
I think we should totally follow http://azure.microsoft.com/en-us/documentation/articles/xplat-cli/ to install Azure CLI.